tests/e2e/backup_restore_suite_test.go

```diff
@@ -409,5 +409,27 @@ var _ = ginkgo.Describe("Backup and restore tests", func() {
 				BackupTimeout:     20 * time.Minute,
 			},
 		}, nil),
+		ginkgo.Entry("MySQL application VSL", ginkgo.FlakeAttempts(flakeAttempts), ApplicationBackupRestoreCase{
```

Comment:
need to label these tests, as they will not work everywhere

```go
ginkgo.Label("aws", "azure", "gcp")
```

Reply:
its a VSL backup, i expect it to work everywhere. Not specific to AWS.

Reply:
VSL only works where there is a velero cloud provider plugin with VSL support. Which is aws, azure, gcp, but not IBM cloud.

Reply:
ah for IBM..sorry i misunderstood.